content/en/docs/Reference/build_raw_images_with_qemu.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Build Raw images with QEMU"
+linkTitle: "Build Raw images with QEMU"
+weight: 5
+description: This article shows how to bring your own image with Kairos, and build a Kairos derivative from scratch using base container images from popular distributions such as Ubuntu, Fedora, openSUSE, etc.
+---
+
+This page provides a reference guide on how to build raw images for Kairos using QEMU. It covers the process of using a default cloud configuration and a script to generate bootable images. The cloud configuration can be customized to suit different use cases.
+
+## Requirements
+
+- `qemu`
+- A Kairos ISO (or a custom built one)
+
+
+## Default Cloud Configuration
+The following is the default `cloud-config` file used for Kairos. You can modify this file as needed to fit your environment:
+
+```yaml
+#cloud-config
+
+hostname: kairos-{{ trunc 4 .MachineID }}
+
+# Automated install block
+install:
+  # Device for automated installs
+  device: "auto"
+  # Reboot after installation
+  reboot: false
+  # Power off after installation
+  poweroff: true
+  # Set to true to enable automated installations
+  auto: true
+
+## Login
+users:
+- name: "kairos"
+  passwd: kairos
+  #lock_passwd: true
+  #ssh_authorized_keys:
+  #- github:mudler
+
+stages:
+  boot:
+  - name: "Repart image"
+    layout:
+      device:
+        label: COS_PERSISTENT
+      expand_partition:
+        size: 0 # all space
+    commands:
+      # grow filesystem if not used 100%
+      - |
+         [[ "$(echo "$(df -h | grep COS_PERSISTENT)" | awk '{print $5}' | tr -d '%')" -ne 100 ]] && resize2fs /dev/disk/by-label/COS_PERSISTENT
+```
+
+Note: `install.poweroff` is set to `true` to power off the machine after installation and `install.auto` is set to `true` to enable automated installations. Both of these settings are **needed** to function properly.
```

Comment:
Rather use an alert for this, so it stands out

Reply:
:+1: done in https://github.com/kairos-io/kairos-docs/pull/355/commits/ec28350769764822ab7c253a7d74d46a8df9f159